### DISCUSSION TITLE
New Format: Fountain (Screenplay)

### DISCUSSION THREAD

**Discussion by Cicciocotroneo** [gsantner/markor#2302] [Ideas] [answered]
Hi. You may probably know there's a syntax named fountain (www.fountain.io) that allows to write screenplays in plain format through a mechanism similar to markdown: it only applies screenwriting formatting rules (spacing, indentation, font and margins) to specific parts of the text. Would it be nice to implement it in markor - but I have no developing skills

**Comment by gsantner** [comment, marked as answer]
If somebody is interested in implementing it, new formats are welcome unless they bloat too much.

**Reply by eugenefischer** [reply]
I might be interested in doing this, actually. Haven't done Android dev before, but I know Java, and would love it if this app supported .fountain. [The regexes are already on GitHub](https://github.com/nyousefi/Fountain/blob/master/Fountain/FountainRegexes.m) so it might not be that difficult.

Is there something you'd recommend looking at to get a sense for what's needed to add a format? I found [this comment](https://github.com/gsantner/markor/discussions/1745#discussioncomment-4065443), but that's from a few years ago, so I don't know if it matches the current state of things.
